system/security/zcrypt/zcrypt.ipp

```diff
@@ -232,13 +232,13 @@ private:
     ZBuffer& generate_key(int len, ZBuffer& keybuf);
 
     void init();
-    virtual int setPublicKey(const char* publickey);
-    virtual int setPrivateKey(const char* privatekey, const char* passphrase);
+    virtual int setPublicKey(const char* publickeyBuff);
+    virtual int setPrivateKey(const char* privatekeyBuff, const char* passphrase);
 
 public:
-    RSAZCryptor();
-    RSAZCryptor(const char* publickey);
-    RSAZCryptor(const char* privatekey, const char* passphrase);
+    RSAZCryptor(unsigned traceLevel = 11);
+    RSAZCryptor(const char* publickey, unsigned traceLevel = 11);
```

Comment:
Seems a little high for a default trace level (especially since at that trace level we dump public and private keys)

Reply:
Changed to 0, which was the legacy behaviour